inductiva/resources/machine_groups.py

```diff
@@ -84,7 +84,7 @@ def _fetch_machine_groups_from_api():
         api = compute_api.ComputeApi(inductiva.api.get_client())
         response = api.list_active_user_instance_groups()
         if len(response.body) == 0:
-            print("No active machine groups found.")
+            print("No active computational resources found.")
```

Comment:
we should remove these print statements in API calls and move them to the consumers.

Reply:
Create an issue for that. [here](https://github.com/inductiva/tasks/issues/89)